test/models/ToolpadEditor.ts

```diff
@@ -144,23 +144,25 @@ export class ToolpadEditor {
     await this.page.mouse.up();
   }
 
+  componentCatalogItem(name: string): Locator {
```

Comment:
nit: maybe `getComponentCatalogItem`?

Reply:
sure, I tried to be consistent with `hierarchyItem` below